Gemfile.lock

```diff
@@ -840,7 +840,7 @@ DEPENDENCIES
   web-console (~> 4.2)
 
 RUBY VERSION
-   ruby 3.1.1p18
+   ruby 3.1.2p20
```

Comment:
Please use the same Ruby version as we have locked at the root `.ruby-version`.

Reply:
I will fix the conflict as well